README.md

```diff
@@ -0,0 +1,89 @@
+<div align="center">
```

Comment:
Not too fan of the centered section, I'd rather keep it simpler with align in left and badges first, similar than what we did here: https://github.com/bonustrack/obyte.js